projects/geo/src/lib/layer/layer-legend/layer-legend.component.ts

```diff
@@ -27,5 +28,25 @@ export class LayerLegendComponent {
   }
   private _legend;
 
-  constructor() {}
+  constructor(private capabilitiesService: CapabilitiesService) {}
+
+  validateItemTitle(item): string {
```

Comment:
I think this method should be named "computeItemTitle". "validateItemTitle" implies that the method may raise an error or return a boolean.